src/artifact_providers/zeus.ts

```diff
@@ -34,14 +34,7 @@ export class ZeusArtifactProvider extends BaseArtifactProvider {
   }
 
   /**
-   * Downloads the given artifact file.
-   *
-   * Downloaded URL are cached during the instance's lifetime, so the same
-   * file is downloaded only once.
-   *
-   * @param artifact An artifact object to download
-   * @param downloadDirectory Directory where downloaded artifact is stored
-   * @returns Absolute path to the saved file
+   * @inheritDoc
```

Comment:
Nice, didn't know about this one 👍

Reply:
You can thank your friends Kamil and Daniel for that one :-)

![image](https://user-images.githubusercontent.com/14812505/75279934-ccf9db80-580c-11ea-9d7f-f371c20b3a56.png)